### ISSUE TITLE
arangodb.connections.max  What does that mean? 

### ISSUE THREAD

**Issue by zhenruyan** [arangodb/arangodb-java-driver#160]
How should this parameter be used correctly?

**Comment by mpv1989** [comment]
This parameter stands for the amount of connections the client (driver) opens to the coordinator/server.
For the protocol VelocyStream (default) the default value is 1.
For HTTP the default value is 20.

Whether or not more or less connection improve the performance depends on your hardware/network and load.

**Comment by zhenruyan** [comment]
Whether you need to manually open or cast the connection

**Comment by mpv1989** [comment]
No, you just use an instance of `ArangoDB`. Whenever you execute any database operation a connection got open until the maximum is reached. If the maximum of connections are open the database operations get balanced between the connections (simple round-robin).

For the default protocol of the driver (VelocyStream) their is no need of multiple connection to get a good performance, since the protocol allows sending multiple requests over one connection without waiting for the response. You can benefit from that if you use the instance of `ArangoDB` from within multiple threads.

For more information about VelocyStream take a look [here](https://www.arangodb.com/2017/08/velocystream-async-binary-protocol/).

**Comment by zhenruyan** [comment]
ok   thank you
